pupil-spa/gen_config.sh

```diff
@@ -1,13 +1,5 @@
 #!/bin/bash
-featureUseHpa=${FEATURE_USE_HPA:-"true"}
```

Comment:
@GuyHarwood we will need to remove the env variable `FEATURE_USE_HPA` from all environments once this gets merged.

Reply:
yes